src/main/java/filter/expression/Qualifier.java

```diff
@@ -210,43 +212,60 @@ public boolean isSatisfiedBy(IModel model, TurboIssue issue, MetaQualifierInfo i
         if (isFalse()) return false;
 
         switch (type) {
-        case ID:
-            return idSatisfies(issue);
-        case KEYWORD:
-            return keywordSatisfies(issue, info);
-        case TITLE:
-            return titleSatisfies(issue);
-        case DESCRIPTION:
-            return bodySatisfies(issue);
-        case MILESTONE:
-            return milestoneSatisfies(model, issue);
-        case LABEL:
-            return labelsSatisfy(model, issue);
-        case AUTHOR:
-            return authorSatisfies(issue);
-        case ASSIGNEE:
-            return assigneeSatisfies(model, issue);
-        case INVOLVES:
-            return involvesSatisfies(model, issue);
-        case TYPE:
-            return typeSatisfies(issue);
-        case STATE:
-            return stateSatisfies(issue);
-        case HAS:
-            return satisfiesHasConditions(issue);
-        case NO:
-            return satisfiesNoConditions(issue);
-        case IS:
-            return satisfiesIsConditions(issue);
-        case CREATED:
-            return satisfiesCreationDate(issue);
-        case UPDATED:
-            return satisfiesUpdatedHours(issue);
-        case REPO:
-            return satisfiesRepo(issue);
-        default:
-            assert false : "Missing case for " + type;
-            return false;
+            case ID :
+                return idSatisfies(issue);
+
```

Comment:
I changed it because it was mentioned in the coding guideline which follows the TEAMMARE guideline. So I thought it would be a good idea to follow it.  The whitespace is not common in the code base